### PR TITLE
cmds/core/mount: add tests

### PR DESCRIPTION
**Pull request by binjip978**

### CODE REVIEW

File: cmds/core/mount/mount.go [hunk 1]
```diff
@@ -85,45 +108,43 @@ func getSupportedFilesystem(originFS string) ([]string, bool, error) {
...
+		for _, p := range c.mountsPath {
 			if b, err := os.ReadFile(p); err == nil {
-				fmt.Print(string(b))
-				os.Exit(0)
+				c.stdout.Write(b)
+				return nil
 			}
 		}
-		log.Fatalf("Could not read %s to get namespace", n)
+		return fmt.Errorf("%w: %v to get namespace", errMountPath, c.mountsPath)
```
Comment (rminnich) [inline cmds/core/mount/mount.go:130]: I've been told by other reviewers that the error should come last, wdyt?
"%q:%w", c.mountsPath, errMountPath)
although tbh in this case I think you did the right thing ...

File: cmds/core/mount/mount.go [hunk 2]
```diff
@@ -28,6 +30,11 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var (
+	errUsage     = errors.New("usage")
+	errMountPath = errors.New("can't read mount path")
```
Comment (rminnich) [inline cmds/core/mount/mount.go:35]: I think we're supposed to avoid contractions in error messages?
can't read mount path
can not read mount path

**Review by rminnich — APPROVED** [review]
This is great. See what you think about my minor comments.

I wonder if at some point this can become a package.

File: cmds/core/mount/mount.go (hunk 2 above)
Comment (binjip978) [reply cmds/core/mount/mount.go]: updated

File: cmds/core/mount/mount.go (hunk 1 above)
Comment (binjip978) [reply cmds/core/mount/mount.go:130]: I think it true, but here I use %w to simplify testing and for nothing else.

**Review by rminnich — APPROVED** [review]

**Pull request merged**